### PR TITLE
integrate api cancel

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/api/run/Run.ts [hunk 1]
```diff
@@ -92,3 +92,12 @@ export async function runResult(data: {
   )
   return response.data
 }
+
+export async function cancelResultApi(data: {
```
Comment (itutu-tienday) [inline frontend/src/api/run/Run.ts:96]: If the workflow is not RUN, please leave the Cancel button disabled.
* If there are any implementation problems, please let me know.

File: frontend/src/store/slice/Pipeline/PipelineActions.ts [hunk 2]
```diff
@@ -87,3 +88,26 @@ export const pollRunResult = createAsyncThunk<
...
+    RunResultDTO,
+    {
+      uid: string
+    },
+    ThunkApiConfig
+>(`${PIPELINE_SLICE_NAME}/cancelResult`, async ({ uid }, thunkAPI) => {
+  const workspaceId = selectCurrentWorkspaceId(thunkAPI.getState())
+  if (workspaceId) {
+    try {
+      const responseData = await cancelResultApi({
```
Comment (itutu-tienday) [inline frontend/src/store/slice/Pipeline/PipelineActions.ts:102]: Please display the message (snackbar) when the process is done.

- Messages
  - On success (code 200):  "Workflow stopped."
  - On error (except code 200): "Failed to stop workflow. Please try again."

**Review by itutu-tienday — CHANGES_REQUESTED** [review]
@sanglevinh 
Please check the indicated items.

File: frontend/src/store/slice/Pipeline/PipelineActions.ts (hunk 2 above)
Comment (sanglevinh) [reply frontend/src/store/slice/Pipeline/PipelineActions.ts:102]: @itutu-tienday  i added it, please check again

File: frontend/src/api/run/Run.ts (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/api/run/Run.ts:96]: @itutu-tienday i fixed it, please check again

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/api/run/Run.ts (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/api/run/Run.ts:96]: @itutu-tienday I have used it, can you check if it aligns with your intention?

Comment (itutu-tienday) [reply frontend/src/api/run/Run.ts:96]: @sanglevinh 
I have checked the operation and it appears to be working as expected.
Are there any concerns in the code as well? If there are no particular concerns, I will approve this fix.

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/api/run/Run.ts (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/api/run/Run.ts:96]: @itutu-tienday I don't see any problem at the moment, if there is, I will fix it later

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**